src/main/java/com/example/student_crud_spring/controller/StudentController.java

```diff
@@ -0,0 +1,87 @@
+package com.example.student_crud_spring.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.student_crud_spring.dto.StudentDto;
+import com.example.student_crud_spring.service.StudentService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/students")
+public class StudentController {
+    private static final Logger logger = LoggerFactory.getLogger(StudentController.class);
```

Comment:
Log errors where needed, in your code this is JooqConfig class. Don`t log each one method.
(In the previous task, logging was to understand the general concept of logging)